calabash/Classes/FranklyServer/LPRouter.m

```diff
@@ -206,11 +206,21 @@ - (BOOL) supportsMethod:(NSString *) method atPath:(NSString *) path {
   if ([[NSThread currentThread] isMainThread]) {
     return [self httpResponseOnMainThreadForMethod:method URI:path];
   } else {
-    __weak typeof(self) wself = self;
-    __block NSObject<LPHTTPResponse> *result = nil;
-    dispatch_sync(dispatch_get_main_queue(), ^{
-      result = [wself httpResponseOnMainThreadForMethod:method URI:path];
-    });
+    // Use performSelectorOnMainThread instead of dispatch_sync because using a dispatch method prevents
```

Comment:
In my opinion, the comment is not helpful.

Can you add a backdoor to the LPTestTarget app and a cucumber that demonstrates that this change allows for backdoors that call dispatch_after?

* https://github.com/calabash/calabash-ios-server/blob/develop/LPTestTarget/AppDelegate.m#L33
* https://github.com/calabash/calabash-ios-server/tree/develop/cucumber/features (add backdoor.feature with a Scenario for this example)